keyboards/dasky/reverb/info.json

```
@@ -0,0 +1,199 @@
+            {"matrix": [1, 2], "x": 42, "y": 14, "flags": 4},
+            {"matrix": [1, 3], "x": 60, "y": 13, "flags": 4},
+            {"matrix": [2, 2], "x": 75, "y": 19, "flags": 4},
+            {"matrix": [2, 3], "x": 91, "y": 24, "flags": 4},
+            {"matrix": [2, 5], "x": 87, "y": 35, "flags": 4},
+            {"matrix": [2, 4], "x": 71, "y": 30, "flags": 4},
+            {"matrix": [1, 5], "x": 56, "y": 24, "flags": 4},
+            {"matrix": [1, 4], "x": 38, "y": 24, "flags": 4},
+            {"matrix": [0, 5], "x": 20, "y": 24, "flags": 4},
+            {"matrix": [0, 4], "x": 5, "y": 22, "flags": 4},
+            {"matrix": [0, 6], "x": 0, "y": 32, "flags": 4},
+            {"matrix": [0, 7], "x": 16, "y": 35, "flags": 4},
+            {"matrix": [1, 6], "x": 34, "y": 35, "flags": 4},
+            {"matrix": [1, 7], "x": 52, "y": 35, "flags": 4},
+            {"matrix": [2, 6], "x": 68, "y": 42, "flags": 4},
+            {"matrix": [2, 7], "x": 83, "y": 46, "flags": 4},
+            {"matrix": [6, 0], "x": 60, "y": 54, "flags": 5},
+            {"matrix": [6, 1], "x": 76, "y": 58, "flags": 5},
+            {"matrix": [6, 2], "x": 96, "y": 64, "flags": 5},
+            {"matrix": [6, 3], "x": 128, "y": 64, "flags": 5},
+            {"matrix": [6, 4], "x": 148, "y": 58, "flags": 5},
+            {"matrix": [6, 5], "x": 164, "y": 54, "flags": 5},
+            {"matrix": [3, 6], "x": 141, "y": 46, "flags": 4},
+            {"matrix": [3, 7], "x": 156, "y": 42, "flags": 4},
+            {"matrix": [4, 6], "x": 172, "y": 35, "flags": 4},
+            {"matrix": [4, 7], "x": 190, "y": 35, "flags": 4},
+            {"matrix": [5, 6], "x": 208, "y": 35, "flags": 4},
+            {"matrix": [5, 7], "x": 224, "y": 32, "flags": 4},
+            {"matrix": [5, 5], "x": 219, "y": 22, "flags": 4},
+            {"matrix": [5, 4], "x": 204, "y": 24, "flags": 4},
+            {"matrix": [4, 5], "x": 186, "y": 24, "flags": 4},
+            {"matrix": [4, 4], "x": 168, "y": 24, "flags": 4},
+            {"matrix": [3, 5], "x": 153, "y": 30, "flags": 4},
+            {"matrix": [3, 4], "x": 137, "y": 35, "flags": 4},
+            {"matrix": [3, 2], "x": 133, "y": 24, "flags": 4},
+            {"matrix": [3, 3], "x": 149, "y": 19, "flags": 4},
+            {"matrix": [4, 2], "x": 164, "y": 13, "flags": 4},
+            {"matrix": [4, 3], "x": 182, "y": 14, "flags": 4},
+            {"matrix": [5, 2], "x": 199, "y": 14, "flags": 4},
+            {"matrix": [5, 3], "x": 216, "y": 10, "flags": 4},
+            {"matrix": [5, 1], "x": 212, "y": 0, "flags": 4},
+            {"matrix": [5, 0], "x": 195, "y": 3, "flags": 4},
+            {"matrix": [4, 1], "x": 178, "y": 3, "flags": 4},
+            {"matrix": [4, 0], "x": 159, "y": 2, "flags": 4},
+            {"matrix": [3, 1], "x": 144, "y": 9, "flags": 4},
+            {"matrix": [3, 0], "x": 129, "y": 13, "flags": 4}
+        ],
+        "led_count": 54
```

`led_count` isnt a valid config option, and it is instead inferred from the length of the `rgb_matrix.layout` array
```suggestion
        ]
```